javascripts/discourse/components/popular-tags.js

```diff
@@ -9,7 +9,25 @@ export default class PopularTags extends Component {
   constructor() {
     super(...arguments);
     const count = this.args?.params?.count || 10;
-    this.topTags = (this.site.get("top_tags") || []).slice(0, count);
+    let tags;
+    const excludedTags = this.args?.params?.excluded_tags || [];
+    const scope_to_category = this.args?.params?.scope_to_category || false;
+
+    if (scope_to_category) {
+      tags = this.site.category_top_tags;
+    } else {
+      tags = this.site.get("top_tags");
+    }
```

Comment:
```suggestion
      const tags = scope_to_category ? this.site.category_top_tags : this.site.top_tags;
```

This might fail prettier, but it should work and won't need to use `let tags`.